src/client/src/ui/blotter/views/blotterView.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { Table, Column, Cell } from 'fixed-data-table';
 import { DateCell, NotionalCell } from './';
-import Dimensions from 'react-dimensions';
+import Dimensions from './../../../system/dimensions';
```

Comment:
why, we don't export an object with a `Dimensions` property but return the function factory for the decorator (HOC)